db/schema.rb

```diff
@@ -14,16 +14,16 @@
 
   create_table "absences", force: :cascade do |t|
     t.integer  "member_id"
-    t.integer  "performanceset_id"
```

Comment:
Are these changes as a result of running the migrations on your system, or are they additional changes that need another migration?